go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/viper v1.8.1
-	github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes v0.3.45
+	github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes v0.3.49
```

Comment:
Could you actually update to `v0.3.58` that one contains the fix for the duplicated assets

Reply:
yep! just did it